priv/protos/persistent.proto

```diff
@@ -19,27 +19,31 @@ message ReadReq {
 	}
 
 	message Options {
-		event_store.client.shared.StreamIdentifier stream_identifier = 1;
+		oneof stream_option {
+			event_store.client.StreamIdentifier stream_identifier = 1;
+			event_store.client.Empty all = 5;
```

Comment:
example of how this works with backwards compatibility: the fields are indexed as their locations in the encoded proto messages, so eventstore can support passing new information to old versions of eventstoredb because the old versions don't know these indices and will fuzz off that data

kinda cool